qdrant_client/local/persistence.py

```diff
@@ -25,7 +25,7 @@ def try_migrate_to_sqlite(location: str) -> None:
     try:
         dbm_storage = dbm.open(str(dbm_path), "c")
 
-        con = sqlite3.connect(str(sql_path))
+        con = sqlite3.connect(str(sql_path), , check_same_thread=False)
```

Comment:
Probably a typo?

Reply:
Yes, it is a typo. 
comma with a blank should be removed.

I missed a commit before merging.